src/main/java/org/dependencytrack/persistence/jdbi/AnalysisDao.java

```diff
@@ -59,7 +61,17 @@ default AnalysisComment makeAnalysisComment(long analysisId, String comment, Str
         return createComment(analysisId, comment, commenter);
     }
 
-    @SqlUpdate("""
+    @SqlQuery("""
+            SELECT *
+            FROM "ANALYSIS"
+            WHERE "COMPONENT_ID" = :componentId
+            AND "VULNERABILITY_ID" = :vulnId
+            LIMIT 1
+            """)
+    @RegisterBeanMapper(Analysis.class)
```

Comment:
It seems you can simply replace this with `@RegisterRowMapper(AnalysisRowMapper.class)` to solve your original issue of some fields not getting mapped correctly?